docs/docs/developer-guides/oracle-set-up-guide.md

```diff
@@ -305,6 +228,89 @@ And enter the details above, as seen in the screenshot and press: “Submit RPC
 
 ![register_offchain_worker](./oracle-set-up-guide/register-offchain-worker.png)
 
+:::tip
```

Comment:
we can add an explanation comment: 200 + (number of tokens prices being submitted -1) thousand PICA